client-js/app/scripts/app.ts

```diff
@@ -23,10 +25,11 @@ window.addEventListener('WebComponentsReady', function() {
     page(e.detail.path);
   });
 
-  var socket = new registry.JsonRpcSocket('ws://localhost:9000/api/socket2');
+  var socket = new clientRpc.JsonRpcSocket('ws://localhost:9000/api/socket2');
+  socket.prefix = "org.labrad."
   var reg = new registry.RegistryServiceJsonRpc(socket);
+  var echo = new echoServer.EchoServiceJsonRpc(socket);
```

Comment:
given that these both share the same socket and may not have the same prefix, it seems wrong to make prefix a property of the socket; rather, I think these services should build the full method string and then just pass that to the socket.